charts/kargo/templates/api/deployment.yaml

```diff
@@ -78,6 +68,11 @@ spec:
             - name: OIDC_CLIENT_ID
               value: {{ .Values.api.oidc.clientID }}
 {{- end }}
+{{- end }}
+{{- if .Values.api.adminAccount.enabled }}
+          envFrom:
+            - secretRef:
+                name: kargo-api
 {{- end }}
```

Comment:
I think we should unconditionally have the kargo-api secret but have our if condition `{{- if .Values.api.adminAccount.enabled }}` in the Secret instead.